tests/playwright/sanity/modules/nested-accordion/nested-accordion.test.ts

```diff
@@ -58,11 +58,7 @@ test.describe( 'Nested Accordion experiment is active @nested-accordion', () =>
 		const page = await browser.newPage();
 		const wpAdmin = new WpAdminPage( page, testInfo );
 
-		await wpAdmin.setExperiments( {
-			container: 'active',
-			'nested-elements': 'active',
-			e_nested_atomic_repeaters: 'active',
-		} );
+		await setup( wpAdmin, { e_nested_atomic_repeaters: 'active' } );
```

Comment:
`nested-elements` is not active by default. removing this will deactivate nested elements.

Reply:
The `setup` function switches the `nested-elements` experiments on

Reply:
I think that we can remove this line, `e_nested_atomic_repeaters` is active by default for sites with core 3.23 and above.

https://github.com/elementor/elementor/blob/main/core/experiments/manager.php#L445

Reply:
I want to make sure that the experiment is on - other tests might reset it